x-pack/plugins/apm/public/components/app/diagnostics/summary_tab/data_streams_status.tsx

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiBadge, EuiFlexGroup, EuiFlexItem, EuiLink } from '@elastic/eui';
+import { APIReturnType } from '../../../../services/rest/create_call_apm_api';
+import { useApmRouter } from '../../../../hooks/use_apm_router';
+import { FETCH_STATUS } from '../../../../hooks/use_fetcher';
+import { useDiagnosticsContext } from '../context/use_diagnostics';
+import { getIsStandardIndexTemplateName } from '../data_stream_tab';
+
+type DiagnosticsBundle = APIReturnType<'GET /internal/apm/diagnostics'>;
+
+export function DataStreamsStatus() {
+  const { diagnosticsBundle, status } = useDiagnosticsContext();
+  const router = useApmRouter();
+  const isLoading = status === FETCH_STATUS.LOADING;
+  const tabStatus = getDataStreamTabStatus(diagnosticsBundle);
+
+  return (
+    <EuiFlexGroup>
+      <EuiFlexItem grow={1}>
+        <EuiFlexGroup justifyContent="flexEnd">
+          <EuiFlexItem grow={false}>
+            {isLoading ? (
```

Comment:
Could this be moved to a specific component? Very similar to the one we have in `summary_tab/apm_integration_package_status.tsx`

Reply:
Good idea. Done.